src/main/org/deidentifier/arx/metric/MetricAECS.java

```diff
@@ -46,7 +46,7 @@
      * Creates a new instance
      */
     protected MetricAECS() {
-        super(false, false);
+        super(true, false, false);
```

Comment:
All utilities, except publisher's payout, have monotonic with generalization set to true.  Super class default is also true.